status-im/status-react
# feature #6239 - showing native currency symbol differently for non-ma…

fixes #6239

### Summary:

Allows us to set different name, displayed symbol and logo icon for native currency of other chains.E.g. we can now display native currency on Ropsten as "Ropsten Ether" with "ETHro" as the symbol as opposed to just "Ether" and "ETH".

The main reason for this functionality is to be able to correctly display native currencies in sidechains. 

### Testing notes:

#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- wallet / transactions
- dapps / app browsing
- networks

### Steps to test:
On each network:
- Send a transaction via chat, wallet and a dapp
- Receive a transaction via chat and wallet
- Check transaction history

On each screen Ether is supposed to be displayed like this:

For mainnet:
Name: Ether
Symbol: ETH
Icon: ethereum icon with round border

For ropsten:
Name: Ropsten Ether
Symbol: ETHRO (or ETHro)
Icon: ethereum icon without round border

For rinkeby:
Name: Rinkeby Ether
Symbol: ETHRI (or ETHri)
Icon: ethereum icon without round border

Places where this is mentioned: main wallet screen, wallet send, request, advanced settings (where you set gas info), chat messages for chat transactions, transaction history. (Let me know if I missed any spots)

status: ready
